### PR TITLE
[CHK-3244] Update open api v2 to include an optional "lang" header

### PR DESCRIPTION
**Pull request by EmanueleBVtech**
#### List of Changes
- Updated open api V1 to include an optional header for language preference
- Updated ecommerce-common reference to 1.26.0 
- after merge with https://github.com/pagopa/pagopa-ecommerce-payment-methods-service/pull/227: The language header is now sent to the 3rd party APIS (npg client)

#### Motivation and Context
This change is part of a bigger task aiming to provide NPG with the language preference as set by the user on the checkout FE. 

#### How Has This Been Tested?
Unit test

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by giovanniberti — DISMISSED** [review]

**Review by giovanniberti — APPROVED** [review]

File: src/main/java/it/pagopa/ecommerce/payment/methods/controller/v1/PaymentMethodsController.java [hunk 1]
```diff
@@ -241,6 +241,7 @@ public Mono<ResponseEntity<CalculateFeeResponseDto>> calculateFees(
     @Override
     public Mono<ResponseEntity<CreateSessionResponseDto>> createSession(
                                                                         String id,
+                                                                        String lang,
```
Comment (pietro-tota) [inline src/main/java/it/pagopa/ecommerce/payment/methods/controller/v1/PaymentMethodsController.java:244]: This language will be used to populate NPG POST order/build language field into another PR?

Comment (EmanueleBVtech) [reply src/main/java/it/pagopa/ecommerce/payment/methods/controller/v1/PaymentMethodsController.java:244]: yes 
here 
https://github.com/pagopa/pagopa-ecommerce-payment-methods-service/pull/227

**Review by pietro-tota — DISMISSED** [review]

File: api-spec/v2/api.yaml [hunk 2]
```diff
@@ -250,4 +250,4 @@ components:
   securitySchemes:
     BearerAuth:
       type: http
-      scheme: bearer
+      scheme: bearer
```
Comment (giovanniberti) [inline api-spec/v2/api.yaml:253]: Stray whitespace?
```suggestion
      scheme: bearer
```

**Review by giovanniberti — APPROVED** [review]

File: api-spec/v2/api.yaml (hunk 2 above)
Comment (EmanueleBVtech) [reply api-spec/v2/api.yaml:253]: yep, didn't notice it. thanks

File: src/main/java/it/pagopa/ecommerce/payment/methods/application/v1/PaymentMethodService.java [hunk 3]
```diff
@@ -377,7 +378,9 @@ public Mono<CreateSessionResponseDto> createSessionForPaymentMethod(
                             orderId, // orderId
                             null, // customerId
                             paymentMethod, // paymentMethod
-                            npgDefaultApiKey // defaultApiKey
+                            npgDefaultApiKey, // defaultApiKey
+                            null,
+                            language
```
Comment (pietro-tota) [inline src/main/java/it/pagopa/ecommerce/payment/methods/application/v1/PaymentMethodService.java:383]:
```suggestion
                            npgDefaultApiKey, // defaultApiKey
                            null, //contractId
                            language //language
```

just for parameter readability

**Review by pietro-tota — DISMISSED** [review]
🚀

**Pull request merged**